<ui version="4.0" >
 <class>xmppConfigDialog</class>
 <widget class="QDialog" name="xmppConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>1</number>
       </property>
       <property name="leftMargin" >
        <number>1</number>
       </property>
       <property name="topMargin" >
        <number>1</number>
       </property>
       <property name="rightMargin" >
        <number>1</number>
       </property>
       <property name="bottomMargin" >
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>1</number>
         </property>
         <property name="leftMargin" >
          <number>1</number>
         </property>
         <property name="topMargin" >
          <number>1</number>
         </property>
         <property name="rightMargin" >
          <number>1</number>
         </property>
         <property name="bottomMargin" >
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Jid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="jidEdit" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordEdit" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Profile Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="profileNameEdit" />
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Personnal Server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="personnalServerEdit" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_5" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="portEdit" />
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="addBtn" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Add >></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deletBtn" >
       <property name="maximumSize" >
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;&lt; Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Profiles :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>xmppConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>xmppConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
